src/components/Footer/Footer.js

```diff
@@ -149,19 +140,34 @@ class Footer extends Component {
             </Row>
             <Row noGutters className="sitemap-container">
               <div>
-                { sitemapLink('/', 'Home', false) }
-                { sitemapLink('/about', 'About', false) }
-                { sitemapLink('/members', 'Members', false) }
-                { sitemapLink('/contact', 'Contact', false) }
+              {routes.map((route: RouteType, index: number) => (
+              <Link
+                eventkey={index}
+                key={index}
+                className='sitemap-link'
+                to={route.path}
+                onClick={this.goUp}
+              >
+                <div className="navbar-btn-legend">
+                  { route.legend }
+                </div>
+              </Link>
+            ))}
                 <div className="mark-text">
                   @2020 RoBorregos
                 </div>
               </div>
             </Row>
             <Row className="row-socialMedia">
-              { sitemapIconButton('https://www.instagram.com/roborregos/', <InstagramIcon style={{ fontSize: this.state.icon_size }} />) }
-              { sitemapIconButton('https://www.facebook.com/RoBorregos/', <FacebookIcon style={{ fontSize: this.state.icon_size }} />) }
-              { sitemapIconButton('https://github.com/RoBorregos/', <GitHubIcon style={{ fontSize: this.state.icon_size - 2, paddingBottom: '0.5vh' }} />) }
+                <a href={'https://www.instagram.com/roborregos/'} className="icon-link">
+                <InstagramIcon style={{ fontSize: this.state.icon_size }} />
+                </a>
+                <a href={'https://www.facebook.com/RoBorregos/'} className="icon-link">
+                <FacebookIcon style={{ fontSize: this.state.icon_size }} />
+                </a>
+                <a href={'https://github.com/RoBorregos/'} className="icon-link">
+                <GitHubIcon style={{ fontSize: this.state.icon_size - 5 }} />
+                </a>
```

Comment:
Add `target='_blank'` to the `<a></a>` to open all the links in a new tab

Reply:
buena call